tecken/libmarkus.py

```diff
@@ -6,8 +6,26 @@
 import time
 from functools import partialmethod
 
+import markus
 from markus import INCR, GAUGE, HISTOGRAM, TIMING
 from markus.backends import BackendBase
+from markus.filters import AddTagFilter
+
+_IS_MARKUS_SETUP = False
+METRICS = markus.get_metrics("tecken")
```

Comment:
this has to be imported and reused everywhere instead of re-calling `markus.get_metrics("tecken")` in order for the AddTagFilter to apply

Reply:
You can specify it as part of the backend configuration, too. We do that in Eliot:

https://github.com/mozilla-services/eliot/blob/82bcae05b11400c72c4299709690357788003bc1/eliot/libmarkus.py#L181

Reply:
I don't see any reference to a host tag in the backend configuration there, except when eliot uses its shared `METRICS` constant the same way this PR does:

https://github.com/mozilla-services/eliot/blob/82bcae05b11400c72c4299709690357788003bc1/eliot/libmarkus.py#L226:
```
        METRICS.filters.append(AddTagFilter(f"host:{hostname}"))
```

Reply:
per slack [@willkg 15 minutes ago](https://mozilla.slack.com/archives/C060L2ZA0DT/p1704306428334699?thread_ts=1704306007.642629&cid=C060L2ZA0DT)
> [@relud](https://mozilla.slack.com/team/U5R388Y5A) on second thought, keep what you have with the way the filter stuff is implemented. i bet i did it this way in eliot because it was hard to verify that the filter kicks in.

Reply:
> this has to be imported and reused everywhere instead of re-calling markus.get_metrics("tecken") in order for the AddTagFilter to apply

Late to the game here, but this is because the tag is tied to that instance of the `MetricsInterface` returned by `markus.get_metrics("tecken")` -- so we have to use the same instance we added the tag to when the app starts up via `_configure_markus`?

Reply:
that is my understanding